schemas/json/block.json

```diff
@@ -552,6 +552,12 @@
 					"isDefault": {
 						"type": "boolean",
 						"default": false
+					},
+					"variations": {
```

Comment:
This isn't working for me when testing this branch - I'm still getting a warning in VSCode that this property isn't allowed when adding it to `block.json` 🤔

Reply:
Sometimes I had that warning too in VSCode when I first check out the branch, but it goes away if I reopen that file. Probably some VSCode cache? 🤔 . Even with the warning it still works though for me? What do you mean not working?